### PR TITLE
Misc changes

### PR DESCRIPTION
**Pull request by mykola-kobets-epam**

### CODE REVIEW

File: src/iam/certmodules/pkcs11/pkcs11.cpp [hunk 1]
```diff
@@ -667,7 +667,7 @@ Error PKCS11Module::FindObject(pkcs11::SessionContext& session, const SearchObje
         templ.EmplaceBack(CKA_LABEL, Array<uint8_t>(labelPtr, filter.mLabel.Size()));
     }
 
-    if (!filter.mType.HasValue()) {
+    if (filter.mType.HasValue()) {
```
Comment (al1img) [inline src/iam/certmodules/pkcs11/pkcs11.cpp:670]: Commit message:  `didn't consider input class` what does it mean?

File: include/aos/common/pkcs11/privatekey.hpp [hunk 2]
```diff
@@ -132,11 +132,6 @@ class PKCS11ECDSAPrivateKey : public crypto::PrivateKeyItf {
     }
 
 private:
-    Error ParseECDSASignature(const Array<uint8_t>& src, Array<uint8_t>& r, Array<uint8_t>& s) const;
```
Comment (al1img) [inline include/aos/common/pkcs11/privatekey.hpp:135]: Commit message: better to write: remove asn1 encoding or similar.

File: src/iam/certmodules/pkcs11/pkcs11.cpp (hunk 1 above)
Comment (mykola-kobets-epam) [reply src/iam/certmodules/pkcs11/pkcs11.cpp:670]: mType value  specifies pkcs11 class like CKO_CERTIFICATE, CKO_PUB_KEY. this value was ignored due to logical error

File: include/aos/common/pkcs11/privatekey.hpp (hunk 2 above)
Comment (mykola-kobets-epam) [reply include/aos/common/pkcs11/privatekey.hpp:135]: ok I will change it.

**Review by al1img — APPROVED** [review]

**Review by MykolaSolyanko — APPROVED** [review]

**Pull request merged**